packages/atomic/src/components/atomic-result-list-v1/atomic-result-table.pcss

```diff
@@ -0,0 +1,54 @@
+@define-mixin atomic-result-table {
+  min-width: 100%;
+  margin: 2rem 0;
+  border-spacing: 0;
+  border-collapse: separate;
```

Comment:
`border-collapse` wasn't needed until now. Its default value is `separate` in the usual context, but in a shadowDOM with Tailwind it seems to be `collapse`.